nipc-openapi/NIPC_REST.yaml

```
@@ -112,7 +100,7 @@ paths:
       parameters:
         - name: id
           in: query
-          description: device ids that need to be filtered
+          description: device or group id that need to be filtered
           required: false
           explode: true
           schema:
```

For comma-separated query parameters, it needs to be defined as such: 
```suggestion
          explode: false
          schema:
            type: array
            items:
              type: string
              format: uuid
              example: 12345678-1234-5678-1234-56789abcdef4
```
This maps to the request: 
`https://localhost/nipc/connectivity/binding?id=12345678-1234-5678-1234-56789abcdef4,12345678-1234-5678-1234-56789abcdef5`

---

updated, pls check

---

Have you pushed the change? I Don't see it.